code/game/objects/items/tools/wirecutters.dm

```diff
@@ -33,19 +33,18 @@
 	var/random_color = TRUE
 	/// List of possible random colors
 	var/static/list/wirecutter_colors = list(
-		"blue" = "#1861d5",
-		"red" = "#951710",
-		"pink" = "#d5188d",
-		"brown" = "#a05212",
-		"green" = "#0e7f1b",
-		"cyan" = "#18a2d5",
-		"yellow" = "#d58c18"
+		COLOR_TOOL_BLUE,
+		COLOR_TOOL_RED,
+		COLOR_TOOL_PINK,
+		COLOR_TOOL_BROWN,
+		COLOR_TOOL_GREEN,
+		COLOR_TOOL_CYAN,
+		COLOR_TOOL_YELLOW,
 	)
 
 /obj/item/wirecutters/Initialize(mapload)
 	if(random_color)
-		var/our_color = pick(wirecutter_colors)
-		set_greyscale(colors=list(wirecutter_colors[our_color]))
+		set_greyscale(colors = list(pick(wirecutter_colors)))
```

Comment:
why was it made like this? player-color editing? vending machines that let you pick their color?

I just want to be sure nothing will break by this.

Reply:
If there has ever been a way to choose which colour you can make your screwdriver/wirecutters, it's not known to me.